docs/README.md

```diff
@@ -13,8 +13,18 @@ This is the documentation of AMS, it's focused on the **0.10.x version.**
 ## How to
 
 - [How to add root key](howto/add_root_key.md)
-- [How to add pagination links](howto/add_pagination_links.md)
 - [Using AMS Outside Of Controllers](howto/outside_controller_use.md)
+- [How to add pagination links](howto/add_pagination_links.md)
+
+#### JSON API Resource `type`
+
+##### Implicit
+
+Resource class name is used, e.g. the type of a `Post` will be `posts` in the resulting json api resource.
+
+##### Explicitly
+
+In the Serializer, declare `type foo`, and `"type": "foo"` will be present in the corresponding resulting json api resource.
```

Comment:
I've been thinking about specializing resources (our current serializers) depending on the adapter, but that breaks the possibility to "just switch adapters" (which arguably is not that useful to start with, but still).